### ISSUE TITLE
Homepage Redesign

### ISSUE THREAD

**Issue by Daltz333** [wpilibsuite/frc-docs#899]
Low Priority, but this looks :fire:
![image](https://user-images.githubusercontent.com/10674555/96400325-6d3ab100-119e-11eb-9144-3d7320362612.png)

**Comment by Starlight220** [comment]
I feel that the multitude of WPI logos is a bit overwhelming. Any chance to either have different logos for each section or forego the logo part altogether?

**Comment by Daltz333** [comment]
It's just placeholders and not a final design at all....

**Comment by Daltz333** [comment]
Blocked by Sphinx-Panels conflicting with Sphinx-Tabs. Nothing in the near-future for resolving this. If Sphinx-Panels adds compatibility with functionality similar to group-tabs, we can begin a migration as it is more maintained.

**Comment by Daltz333** [comment]
*very* rough sketch in MSPaint of my idea. Jotting it down here for further R&D

![image](https://user-images.githubusercontent.com/10674555/104799101-45a5b580-579a-11eb-81e9-dc2bb340627a.png)

**Comment by Daltz333** [comment]
A partial implementation 
![image](https://user-images.githubusercontent.com/10674555/106377250-d3240080-6369-11eb-8674-98ca0c132d74.png)

**Comment by sciencewhiz** [comment]
Fixed by #1390?
